java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S1212_java.html

```diff
@@ -6,6 +6,4 @@
   <li> forbid access to <code>java.util.Vector</code>, <code>java.util.Hashtable</code> and <code>java.util.Enumeration</code> from any classes </li>
   <li> forbid access to <code>java.sql.**</code> from <code>**.ui.**</code> and <code>**.web.**</code> classes </li>
 </ul>
-<h2>Deprecated</h2>
-<p>This rule is deprecated, and will eventually be removed.</p>
```

Comment:
Why remove this ? rule is actually deprecated : https://jira.sonarsource.com/browse/RSPEC-1212